compiler/rustc_error_codes/src/lib.rs

```diff
@@ -441,7 +441,7 @@ E0695: 0695,
 E0696: 0696,
 E0697: 0697,
 E0698: 0698,
-E0699: 0699,
+E0699: 0699, // REMOVED: merged into generic inference var error
```

Comment:
You can comment it and move it to the commented list below (but keep your comment).

Reply:
no I cannot 😆 I tried this first.

Reply:
Wait really? O.o

Dark magic going on...

Reply:
The comment at the top of the file says to not remove things anymore

Reply:
Hum... [E0744](https://github.com/rust-lang/rust/blob/master/compiler/rustc_error_codes/src/lib.rs#L677) is commented, its file [still exists](https://github.com/rust-lang/rust/blob/master/compiler/rustc_error_codes/src/error_codes/E0744.md) and you can get it from `rustc --explain E0744`, so I'm surprised. Oh well, as long as CI passes...